opera/validate_test.go

```diff
@@ -238,10 +242,12 @@ func TestBlocksRulesValidation_DetectsIssues(t *testing.T) {
 }
 
 func TestBlocksRulesValidation_AcceptsValidRules(t *testing.T) {
+	const maxEmptyBlockSkipPeriod = inter.Timestamp(minEmptyBlockSkipPeriod * time.Second)
```

Comment:
nit: now that `minEmptyBlockSkipPeriod` is in seconds, this conversion is misleading; it produces a duration of 4 billion seconds.